File: 프로그래머스/unrated/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.js
function solution(n) {
  let answer = 0
  
  for (let i =0; i <= n ; i++){
      if(n%2!==0&&i%2!==0){
          answer+=i
          // console.log(i)
      }else if(n%2===0&&i%2===0){
          answer += i*i         
      }
  }
    return answer
}

File: 프로그래머스/unrated/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for (int i =0; i <= n ; i++){
            if(n%2!=0&&i%2!=0){
                answer+=i;
            }else if(n%2==0&&i%2==0){
                answer += i*i;
            }
        }
    return answer;
    }
}